src/test/regress/expected/multi_size_queries.out

```diff
@@ -0,0 +1,106 @@
+--
+-- MULTI_SIZE_QUERIES
+--
+-- Test checks whether size of distributed tables can be obtained with citus_table_size.
+-- To find the relation size and total relation size citus_relation_size and 
+-- citus_total_relation_size are also tested.
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1390000;
+ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1390000;
+-- Tests on distributed table with replication factor > 1
+SELECT citus_table_size('lineitem_hash_part');
+ERROR:  cannot calculate the size because replication factor is greater than 1
+SELECT citus_relation_size('lineitem_hash_part');
+ERROR:  cannot calculate the size because replication factor is greater than 1
+SELECT citus_total_relation_size('lineitem_hash_part');
+ERROR:  cannot calculate the size because replication factor is greater than 1
+VACUUM (FULL) customer_copy_hash;
+-- Tests on distributed tables with streaming replication.
+SELECT citus_table_size('customer_copy_hash');
+ citus_table_size 
+------------------
+           548864
+(1 row)
+
+SELECT citus_relation_size('customer_copy_hash');
+ citus_relation_size 
+---------------------
+              548864
+(1 row)
+
+SELECT citus_total_relation_size('customer_copy_hash');
+ citus_total_relation_size 
+---------------------------
+                   1597440
+(1 row)
+
+CREATE INDEX index_1 on customer_copy_hash(c_custkey);
+NOTICE:  using one-phase commit for distributed DDL commands
+HINT:  You can enable two-phase commit for extra safety with: SET citus.multi_shard_commit_protocol TO '2pc'
+VACUUM (FULL) customer_copy_hash;
+-- Tests on distributed table with index.
+SELECT citus_table_size('customer_copy_hash');
+ citus_table_size 
+------------------
+           548864
+(1 row)
+
+SELECT citus_relation_size('customer_copy_hash');
+ citus_relation_size 
+---------------------
+              548864
+(1 row)
+
+SELECT citus_total_relation_size('customer_copy_hash');
+ citus_total_relation_size 
+---------------------------
+                   2646016
```

Comment:
I have checked this with another table. Size increases dramatically when we add an index to distributed table. So, I think it is expected.